wandb/sdk/launch/agent/agent.py

```diff
@@ -169,17 +169,26 @@ def loop(self) -> None:
                         break
                 if not job:
                     time.sleep(AGENT_POLLING_INTERVAL)
+                    agent_response = self._api.get_launch_agent(
```

Comment:
Shouldn't this be a `launch_agent_heartbeat` mutation that updates the agent heartbeat like with sweep agents?

Reply:
It should also retrieve the information about whether to keep polling.

Reply:
discussed, heartbeats are only updated on status update, leaving it for now